### PR TITLE
Add Snyk

### PR DESCRIPTION
**Pull request by sjparkinson**
This is the result of running:

* `snyk wizard`
* and `npm audit fix`

I needed to add an update of npm as the current version was failing to install express-handlebars.

### CODE REVIEW

**Comment by sjparkinson** [comment]
https://github.com/Financial-Times/n-handlebars/issues/68

**Comment by sjparkinson** [comment]
https://github.com/Geo1088/wix-maybe/commit/d975ff620cbee77d38043684123a7d276bf31af1

Look at applying the above change.

**Comment by sjparkinson** [comment]
Try node-sass 4.6.1 which allows minor version changes of the request package.

File: package.json [hunk 1]
```diff
@@ -109,29 +111,27 @@
     "@quarterto/chai-dom-equal": "^1.6.0",
     "@quarterto/chai-selector": "^1.3.0",
     "@quarterto/emphasize-monokai-sheet": "^1.3.0",
-    "@quarterto/gaussian": "^1.0.0",
+    "@quarterto/gaussian": "^1.4.0",
```
Comment (quarterto) [inline package.json]: my gist to npm script doesn't actually do semver 😬 it just increments the minor version on each commit. i don't think this version is backwards compatible. then again, it's only used in [a script to analyse the benchmark results](https://github.com/Financial-Times/google-amp/blob/e62a88e01995e607c5724e4e3461f1447b2bfe3c/scripts/dev-utils/analyse-log.js#L5), which i haven't run in about 18 months, and we should probably do speedcurve to it instead

File: package.json [hunk 2]
```diff
@@ -109,29 +111,27 @@
...
     "@quarterto/gaussian-percentile": "^1.0.0",
     "@quarterto/heroku-config-to-env": "^1.3.1",
     "@quarterto/mocha-describe-folder": "^1.3.0",
-    "@quarterto/symbol-logger": "^1.6.0",
-    "amphtml-validator": "^1.0.10",
+    "@quarterto/symbol-logger": "^1.16.0",
```
Comment (quarterto) [inline package.json]: again, absolutely no guarantees this is backwards compatible. it's only used in a [utility script](https://github.com/Financial-Times/google-amp/blob/e62a88e01995e607c5724e4e3461f1447b2bfe3c/scripts/dev-utils/selector-example.js#L13)

File: package.json [hunk 3]
```diff
@@ -44,7 +46,6 @@
     "@quarterto/kinesis-decoded": "^1.0.0",
     "@quarterto/log-promise": "^1.1.0",
     "@quarterto/log-task": "^1.3.0",
-    "@quarterto/mocha-describe-folder": "^1.4.0",
     "@quarterto/package-version-github-release": "^1.2.0",
     "@quarterto/package-version-sentry-release": "^1.1.0",
     "@quarterto/post-sass": "^1.3.0",
```
Comment (quarterto) [inline package.json]: `post-sass` isn't used anywhere anymore and should die in a fire

File: package.json (hunk 2 above)
Comment (sjparkinson) [reply package.json]: Seems to work 👌

**Pull request merged**